test/batch-submitter/transaction-batch-submitter.spec.ts

```diff
@@ -282,3 +297,66 @@ describe('TransactionBatchSubmitter', () => {
     })
   })
 })
+
+describe('TransactionBatchSubmitter to Ganache', () => {
+  let signer
+  const server = ganache.server({
+    default_balance_ether: 420,
+    blockTime: 2_000
+  })
+  const provider = new Web3Provider(ganache.provider())
+
+  before(async () => {
+    await server.listen(3001)
+    signer = await provider.getSigner()
+  })
+
+  after(async () => {
+    await server.close()
+  })
+
+  // only write test for getReceiptWithResubmission
```

Comment:
dead comment

Reply:
It may be worth adding a comment here generally explaining how this test works

Reply:
^will do that!